src/TwigRendererFactory.php

```diff
@@ -28,9 +29,7 @@
  */
 class TwigRendererFactory
 {
-    /**
-     * @throws Exception\InvalidConfigException for invalid config service values.
-     */
+
```

Comment:
Please revert - this method still throws that exception, via the `getEnvironment()` call.